mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -207,37 +207,68 @@ function removeTrack() {
 }
 
 function clearQueue() {
-    mopidy.playback.stop();
-    resetSong();
-    mopidy.tracklist.clear();
-    resetSong();
+    mopidy.playback.stop().then(
```

Comment:
If we want we can safely omit the `stop`command since clearing does that for us.